test/new-e2e/tests/npm/ec2_1host_wkit_test.go

```diff
@@ -35,8 +34,8 @@ func TestEC2VMWKitSuite(t *testing.T) {
 func (v *ec2VMWKitSuite) SetupSuite() {
 	v.BaseSuite.SetupSuite()
 
-	v.Env().RemoteHost.MustExecute("Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))")
-	v.Env().RemoteHost.MustExecute("C:\\ProgramData\\chocolatey\\choco install /y apache-httpd --params '\"/noService\"'")
```

Comment:
Did we try adding some retries here? That seems like the clearer/easier fix. Especially if these   commands are idempotent

Reply:
I can't add retry in chocolatey and windows folks told me that chocolatey service is unstable

Reply:
@leeavital chocolatey == bad.
Among other things, they throttle accesses.  So if we have a burst of build pipelines, some will just fail because chocolatey will reject the accesses.  This, while adding some overhead to us, is unfortunately much more reliable.